src/main/resources/requests/accountManagementRequests.ts

```diff
@@ -127,6 +127,24 @@ export class AccountManagementRequests {
         return null;
     }
 
+    public async sendMediaApplicationRejectionEmail(
+        applicantId: string,
+        reasons: string
+    ): Promise<MediaAccountApplication | null> {
+        try {
+            const response = await accountManagementApi.post(`/application/reject/${applicantId}`, { reasons });
```

Comment:
Sorry - I have fixed the acc man endpoint. I'm in partial agreement but there is no existing route straight from frontend -> pub servs so Chris thought it would be best if we put it all in the same flow.